frontend/src/components/beta/accounts/AccountActivityView.tsx

```diff
@@ -0,0 +1,485 @@
+            <TableElement>
+              {subindex === 0 ? (
+                <>
+                  {item.direction === "inbound" ? "<<<" : ">>>"}
+                  {item.involvedAccountId ? (
+                    <Link href={getAccountLink(item.involvedAccountId)}>
+                      <a>{shortenString(item.involvedAccountId)}</a>
+                    </Link>
+                  ) : (
+                    "system"
+                  )}
+                </>
+              ) : null}
+            </TableElement>
+            <TableElement>
+              <ActivityItemAction action={subaction} />
+              {item.action.parentAction ? (
+                <>
+                  <hr />
+                  <ActivityItemTitle>Caused by receipt:</ActivityItemTitle>
+                  <ActivityItemAction action={item.action.parentAction} />
+                </>
+              ) : null}
+              {childrenActions.length !== 0 ? (
+                <>
+                  <hr />
+                  <ActivityItemTitle>Children receipts:</ActivityItemTitle>
+                  {childrenActions.map((childAction, index) => (
+                    <ActivityItemAction key={index} action={childAction} />
+                  ))}
+                </>
+              ) : null}
+            </TableElement>
+            <TableElement>
+              {!isDeltaAmountZero && subindex === 0 ? (
+                <Amount
+                  direction={isDeltaAmountPositive ? "income" : "outcome"}
+                >
+                  {isDeltaAmountPositive ? "+" : "-"}
+                  <NearAmount
+                    amount={absoluteDeltaAmount.toString()}
+                    decimalPlaces={2}
+                  />
+                </Amount>
+              ) : (
+                "—"
+              )}
+            </TableElement>
+            <TableElement>
+              {subindex === 0 ? (
+                <>
+                  <LinkPrefix>
+                    {"transactionHash" in item.action
+                      ? item.action.receiptId
+                        ? "RX"
+                        : "TX"
+                      : "BL"}
+                  </LinkPrefix>
+                  <Link href={getActionLink(item.action)}>
+                    {shortenString(
+                      "blockHash" in item.action
+                        ? item.action.blockHash
+                        : item.action.transactionHash
+                    )}
+                  </Link>
+                  <CopyWrapper>
+                    <CopyToClipboard
+                      text={
+                        "blockHash" in item.action
+                          ? item.action.blockHash
+                          : item.action.transactionHash
+                      }
+                    />
+                  </CopyWrapper>
+                </>
+              ) : null}
+            </TableElement>
+            <DateTableElement>
+              {subindex === 0
+                ? moment
```

Comment:
I guess we don’t need `moment` here as we switched to another library

Reply:
That's what I found as well after rebase :)